client/extensions/woocommerce/app/settings/shipping/save-button.js

```diff
@@ -61,11 +79,11 @@ class ShippingSettingsSaveButton extends Component {
 
 		if ( finishedInitialSetup ) {
 			return wcsEnabled
-				? <Button onClick={ this.save } primary>{ translate( 'Save' ) }</Button>
+				? <Button onClick={ this.save } primary busy={ isSaving } disabled={ isSaving }>{ translate( 'Save' ) }</Button>
```

Comment:
Yep, I think that's okay. The issue with the Save button in each of the separate sections is that people may make changes to two sections and then be unsure if they have to save each section individually, or maybe forget what section they changed. Its an issue we've ran across in the current implementation of the individual save buttons on the settings pages in calypso. (Right now they act as global save buttons, but they appear on every individual sections and users don't know if they have to click each one or not)